### ISSUE TITLE
Cleanup

### ISSUE THREAD

**Issue by fwaibl** [liedllab/surface_analyses#17]
There are some things that we might want to change at some point:

* We currently depend on the gisttools code, although we don't do anything GIST-related. We might want to remove this dependency (or keep it as an optional dependency for the surface calculation)
* We could remove the CDR annotation feature. This does not seem to fit with the other features, and can be done by the user after the run.

**Comment by vhoer** [comment]
- [ ] Currently, the hydrophobicity script takes an MD trajectory as input whereas the electrostatics script requieres a PDB file. We should probably think of a consistent way to handle this, I'd say use PDB files as input and add a wrapper for trajectories.

**Comment by fwaibl** [comment]
I agree that the current input loading is very inconsistent. However, I like trajectories as input since the hydrophobicity scripts are very fast, and adding a wrapper where each frame has its own process would make the whole run much slower (importing all the modules takes a lot of time).

Another alternative would be use exclusively mdtraj's input functions. In that case we could support trajectories and PDBs at the same time. Currently, this is not the case, since the electrostatics script also uses the PDB input file directly, which makes the whole situation a lot messier...

**Comment by vhoer** [comment]
We could instead also use mdtrajs input functions for the electrostatics and convert to PDB files if a trajectory is supplied. While that would be slow, at least both scripts would be consistent. Should be fine with aditional warnings to the user if too many frames are supplied.

**Comment by fwaibl** [comment]
I'll check what the PDB file is actually used for. I don't care so much about speed for the electrostatics script, since the grid operations are quite slow anyway, especially if we call APBS.
